proposals/IpfsGatewayHttpApi.md

```diff
@@ -0,0 +1,134 @@
+# IPFS Gateway HTTP API
```

Comment:
Perhaps this would be less vague?

```suggestion
# Writable HTTP Gateway
```

Reply:
Writeable is only part of it. I actually think the first 70% of value will come from readable/queryable DAGs